rdf4j/src/main/java/com/inrupt/client/rdf4j/RDF4JBodyHandlers.java

```diff
@@ -41,45 +41,83 @@
  */
 public final class RDF4JBodyHandlers {
 
+    private static Boolean isSuccess(final Response.ResponseInfo responseInfo) {
+        return responseInfo.statusCode() < 300;
+    }
+
+    private static Model responseToModel(final Response.ResponseInfo responseInfo) {
+        return responseInfo.headers().firstValue("Content-Type")
+                .map(RDF4JBodyHandlers::toRDF4JFormat).map(format -> {
+                    try (final InputStream stream = new ByteArrayInputStream(responseInfo.body().array())) {
+                        return Rio.parse(stream, responseInfo.uri().toString(), format);
+                    } catch (final IOException ex) {
+                        throw new UncheckedIOException(
+                                "An I/O error occurred while data was read from the InputStream", ex);
+                    }
+                })
+                .orElseGet(() -> new DynamicModelFactory().createEmptyModel());
+    }
+
     /**
      * Populate a RDF4J {@link Model} with an HTTP response.
      *
      * @return an HTTP body handler
+     * @deprecated
```

Comment:
Point to new like in Jena.

Reply:
Fixed with https://github.com/inrupt/solid-client-java/pull/1162/commits/bf051a11728bddf082be51e988e7cb6cef2d7506